src/terra-dev-site/IssueForm/Helper.jsx

```diff
@@ -0,0 +1,117 @@
+import PropTypes from 'prop-types';
+import Packages from './Packages.json';
+
+// Prop types for all components.
+const propTypes = {
+  /**
+   * List of packages from terra clinical, core, and framework.
+   */
+  packageList: PropTypes.arrayOf(PropTypes.string),
+  /**
+   * Functions that set state with the provided field values.
+   */
+  setContext: PropTypes.func,
+  setDescription: PropTypes.func,
+  setEnvironment: PropTypes.func,
+  setExpected: PropTypes.func,
+  setIssue: PropTypes.func,
+  setMentions: PropTypes.func,
+  setPackage: PropTypes.func,
+  setSolution: PropTypes.func,
+  setSteps: PropTypes.func,
+  setTitle: PropTypes.func,
```

Comment:
lets add comments for each of these

Reply:
Changed: [5eff56a](https://github.com/cerner/terra-ui/pull/151/commits/5eff56a5761ea4b8acbfe97a933e639186f91e48)